app/experimenter/nimbus-ui/src/components/MonitoringLink/open-external.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" viewBox="0 0 8 8" xmlns="http://www.w3.org/2000/svg"><path clip-rule="evenodd" d="m1.71278 0h.57093c.31531 0 .57092.255837.57092.571429 0 .315591-.25561.571431-.57092.571431h-.57093c-.31531 0-.57093.25583-.57093.57143v4.57142c0 .3156.25562.57143.57093.57143h4.56741c.31531 0 .57093-.25583.57093-.57143v-.57142c0-.3156.25561-.57143.57092-.57143.31532 0 .57093.25583.57093.57143v.57142c0 .94678-.76684 1.71429-1.71278 1.71429h-4.56741c-.945943 0-1.71278-.76751-1.71278-1.71429v-4.57142c0-.946778.766837-1.71429 1.71278-1.71429zm5.71629 0c.23083.0002686.43879.13963.52697.353143.02881.069172.04375.143342.04396.218286v2.857141c0 .31559-.25561.57143-.57093.57143-.31531 0-.57092-.25584-.57092-.57143v-1.47771l-1.88006 1.88171c-.14335.14855-.35562.20812-.55523.15583-.19962-.0523-.35551-.20832-.40775-.40811-.05225-.19979.00727-.41225.15569-.55572l1.88006-1.88171h-1.47642c-.31531 0-.57093-.25584-.57093-.571431 0-.315592.25562-.571429.57093-.571429z" fill="#5e5e72" fill-rule="evenodd"/></svg>
```

Comment:
Was confused because I didn't see this used anywhere in this PR. It looks like it was used in `LinkExternal` stories only and not in the actual component. Perhaps you could add it to the `LinkExternal` component? Seems misleading to me that it's only in a story.

Reply:
oh whoops sorry this was just duplicated accidentally